bundle/edu.gemini.sp.vcs/src/test/scala/edu/gemini/sp/vcs/diff/MergeCorrectionSpec.scala

```diff
@@ -21,6 +21,8 @@ import Scalaz._
 class MergeCorrectionSpec extends Specification {
   import NodeDetail._
 
+  val lifespanId: LifespanId = LifespanId.random
```

Comment:
What's the is the `LifespanId` of a `MergeNode` in general?
I was wondering about the use of random here but this is just for the test, right